pkg/agent/types/node_config.go

```diff
@@ -27,12 +28,23 @@ const (
 type GatewayConfig struct {
 	IP   net.IP
 	MAC  net.HardwareAddr
-	Name string
+	Link string
+}
+
+func (g *GatewayConfig) String() string {
+	return fmt.Sprintf("Name %s: IP %s, MAC %s", g.Link, g.IP, g.MAC)
 }
 
 type NodeConfig struct {
-	Bridge  string
-	Name    string
-	PodCIDR *net.IPNet
-	*GatewayConfig
+	Bridge         string
+	Name           string
+	PodCIDR        *net.IPNet
+	NodeIPAddr     *net.IPNet
+	NodeDefaultDev string
```

Comment:
Do we still need this NodeDefaultDev? If not could we remove it?

Reply:
done